docker-compose-workflows-with-temporal.yml

```diff
@@ -155,6 +155,10 @@ services:
     environment:
       - DB=postgresql
       - DYNAMIC_CONFIG_FILE_PATH=config/dynamicconfig/development-sql.yaml
+      # To enable TLS between temporal and external postgres, set both below variables to true
+      - SQL_TLS_ENABLED=false
+      - SQL_TLS=false
+      - SQL_TLS_DISABLE_HOST_VERIFICATION=true
```

Comment:
SQL_TLS shouldn't be needed? and SQL_HOST_VERIFICATION should be used instead?

https://github.com/temporalio/temporal/blob/main/docker/config_template.yaml#L93-L99

https://community.temporal.io/t/ssl-postgres-and-temporal/4216

Reply:
(we can also add both here, in case some service uses sql tool)

Reply:
Currently we need to set both because of https://github.com/temporalio/temporal/issues/2621 (the temporal docker image uses the temporal-sql-tool to setup db schemas) 😢 
ah great catch -- looks like there is also a mismatch for HOST_VERIFICATION so need to duplicate that one too